core/src/main/java/hudson/init/InitMilestone.java

```diff
@@ -100,7 +100,7 @@ public static TaskBuilder ordering() {
         TaskGraphBuilder b = new TaskGraphBuilder();
         InitMilestone[] v = values();
         for (int i=0; i<v.length-1; i++)
-            b.add(null, Executable.NOOP).requires(v[i]).attains(v[i+1]);
+            b.add("ORDER: " + v[i] + " -> " + v[i+1], Executable.NOOP).requires(v[i]).attains(v[i+1]);
```

Comment:
What is this `"ORDER: "` stuff? Does it have an effect on the execution, or is it just a logging thing?

If it has an effect on execution, then I think maybe a comment would help others make sense of it?